releasenotes-builder/src/main/java/com/github/checkstyle/MainProcess.java

```diff
@@ -103,6 +108,12 @@ public static List<String> validateNotes(Multimap<String, ReleaseNotesMessage> r
         final String errorEnding = "Please correct release number by running https://github.com/"
             + "checkstyle/checkstyle/actions/workflows/bump-version-and-update-milestone.yml";
 
+        if (releaseVersion.split("\\.").length != RELEASE_DIGIT_COUNT) {
+            errors.add("Release number should have 3 digits separated by '.' symbols");
+        }
+        if (StringUtils.countMatches(releaseVersion, ".") != RELEASE_DIGIT_COUNT - 1) {
```

Comment:
These 2 seem to be the same thing, counting periods. Validation should be regular expression matching `\d+\.\d+\.\d+`.

Reply:
done.